qCC/ccCommandLineCommands.cpp

```diff
@@ -1767,6 +1680,45 @@ bool CommandSubsample::process(ccCommandLineInterface& cmd)
 	return true;
 }
 
+bool CommandSubsample::ReplaceAndSaveCloud(ccCommandLineInterface& cmd, CLCloudDesc& orignalDesc, ccPointCloud *newPointCloud, const SubsamplingParams& params) {
```

Comment:
```suggestion
bool CommandSubsample::ReplaceAndSaveCloud(ccCommandLineInterface& cmd, CLCloudDesc& orignalDesc, ccPointCloud* newPointCloud, const SubsamplingParams& params) {
```

Reply:
And maybe you could pass a reference to a 'newPointCloud' pointer  (ccPointCloud*& newPointCloud) so as to set it to nullptr after you delete it. So that the function caller has an updated (null) pointer.